systemlink-notebook-datasource/src/QueryEditor.tsx

```diff
@@ -180,7 +173,7 @@ export class QueryEditor extends PureComponent<
   }
 
   render() {
-    const query = defaults(this.props.query, defaultQuery);
+    const query = { ...defaultQuery, ...this.props.query };
```

Comment:
I had to change these places where we were mutating the component's props. For some reason this was only causing errors when used in the variable editor.